tools/VSCODE.md

```diff
@@ -0,0 +1,9 @@
+# Visual Studio Code
+
+Visual Studio Code is the popular open source development environment from Microsoft. Available at [code.visualstudio.com](https://code.visualstudio.com/)
+
+## Extensions
```

Comment:
Do we still use stylelint? Should that be on this list?

Reply:
We do, but haven't actually had time to fix the style issues an re activate in the widgets repo yet.

Reply:
Is that a reason not to encourage folks to install the extension?

Reply:
Duh, just found the `/linters` folder, will update the PR to point to those